cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,36 @@
+// const EMAIL = Cypress.env("USER_EMAIL");
+// const PASSWORD = Cypress.env("USER_PASSWORD");
+// const USER_NAME = Cypress.env("USER_NAME");
```

Comment:
It's a shame that this didn't work out, do these come back undefined?

Reply:
It did work locally! It also worked in automated testing in workflow. 

But I did not know how it will be in the remote environment, since the .env file will not be there. 

I stored those credentials in the repo setting/secrets. How can I use those secret variables in actual test files, or cypress.config.js? If my question make sense?

Reply:
There is no need to test on the remote environment since this is done prior to deployment in actions :)

Reply:
Oh yeah...but when you clone this repo, .env file will not be there and therefore you cannot see if it works or not??

Reply:
Then it's my responsibility to populate the env file myself and use my own test account

Reply:
Got it, thanks!
I'll just add little a note in the README.